resources/eventing/charts/nats/templates/00-prereqs.yaml

```diff
@@ -0,0 +1,130 @@
+{{- if .Values.install.enabled }}
+
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: nats-operator
+  # Change to the name of the namespace where to install NATS Operator.
+  # Alternatively, change to "nats-io" to perform a cluster-scoped deployment in supported versions.
+  namespace: {{ .Values.namespace }}
```

Comment:
Helm labels are missing here. This results in an upgrade problem (from eventing without nats => eventing with nats):

```
Error: UPGRADE FAILED: rendered manifests contain a resource that already exists. Unable to continue with update: ServiceAccount "nats-operator" in namespace "nats" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "eventing"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "eventing"
```

Reply:
This will be solved in the follower issue:
https://github.com/kyma-project/kyma/issues/9969